src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -175,18 +175,42 @@ public void visitJavadocToken(DetailNode ast) {
             final List<DetailNode> textNodes = getAllNewlineNodes(ast);
             for (DetailNode newlineNode : textNodes) {
                 final DetailNode textNode = JavadocUtil.getNextSibling(newlineNode);
-                if (textNode.getType() == JavadocTokenTypes.TEXT) {
-                    final String text = textNode.getText();
-                    if (!CommonUtil.isBlank(text.trim())
-                            && (text.length() <= offset
-                                    || !text.substring(1, offset + 1).trim().isEmpty())) {
-                        log(textNode.getLineNumber(), MSG_KEY, offset);
-                    }
+                if (textNode.getType() == JavadocTokenTypes.TEXT && isViolation(textNode)) {
+                    log(textNode.getLineNumber(), MSG_KEY, offset);
                 }
             }
         }
     }
 
+    /**
+     * Checks if a text node meets the criteria for a violation.
```

Comment:
Maybe a bit more detail here.

Reply:
Ok, added some description on what exactly the algorithm is checking for in the text.

Reply:
Resolved.